setup.py

```diff
@@ -992,7 +992,7 @@ def main():
         if os.path.exists(triton_pin_file):
             with open(triton_pin_file) as f:
                 triton_pin = f.read().strip()
-                extras_require['dynamo'] = ['torchtriton==2.0.0+' + triton_pin[:10], 'jinja2']
+                extras_require['dynamo'] = ['pytorch-triton==2.0.0+' + triton_pin[:10], 'jinja2']
```

Comment:
@malfet Sorry if this is not the right place to ask this.

Does the following command install `pytorch-triton`?

```
pip3 install --pre torch --extra-index-url https://download.pytorch.org/whl/nightly/cpu
```

Looks like it doesn't because it's in `extras_require`.

Reply:
However, the following command does:

```
pip3 install --pre torch[dynamo] --extra-index-url https://download.pytorch.org/whl/nightly/cpu
```